formula/src/main/java/com/instacart/formula/FormulaRuntime.kt

```diff
@@ -15,11 +15,15 @@ class FormulaRuntime<Input : Any, Output : Any>(
     private val onOutput: (Output) -> Unit,
     private val onError: (Throwable) -> Unit,
     private val isValidationEnabled: Boolean = false,
-    private val inspector: Inspector? = null,
+    inspector: Inspector? = null,
 ) : ManagerDelegate {
     private val implementation = formula.implementation()
     private var manager: FormulaManagerImpl<Input, *, Output>? = null
-    private var hasInitialFinished = false
```

Comment:
This is not needed anymore - previously it was used to ensure that on formula start, we wait for all actions to start running before emitting a value. Currently, it's done within `FormulaManagerImpl`.